### DISCUSSION TITLE
v0.7.1 - Automation Demo

### DISCUSSION THREAD

**Discussion by saddam213** [saddam213/OnnxStack#24] [General]
Demo release of the automation(batch) features, debug mode so heavy logging
@Amin456789

<hr /><em>This discussion was created from the release <a href='https://github.com/saddam213/OnnxStack/releases/tag/v0.7.1'>v0.7.1 - Automation Demo</a>.</em>

**Comment by Amin456789** [comment]
uh nice! been waiting for this! thank u! im gonna test it now and report back

**Reply by saddam213** [reply]
there was a bug, I have uploaded a new version :)

**Reply by Amin456789** [reply]
nice!
btw i cant import the models i had, it seems i have to choose a vae by force, none of them has vae so i cant test it anymore

**Reply by saddam213** [reply]
that was the fix I just uploaded :)

**Reply by Amin456789** [reply]
oh hehe cool!

**Comment by Amin456789** [comment]
@saddam213
tested with both lcm and chilloutmix [sd 1.5] models, chose automation with seed type and batch of 4 i tested, it worked perfectly!
best ui ever! deleted all of my sd folders just to use this one forever
thanks man :D

**Comment by Amin456789** [comment]
actually Adam. debug mode is somehow great! i mean the logging, it shows us very good information such as how long every steps took for every image, u should add this as default i guess.
